static/sass/_v1_pattern_strip.scss

```diff
@@ -39,6 +43,12 @@
   }
 }
 
+@mixin ubuntu-p-strip-white {
```

Comment:
Strips are white by default - why is this needed? toggling it on/off makes no difference to visual of the page?